libcst/_typed_visitor.py

```diff
@@ -0,0 +1,2177 @@
+# This source code is licensed under the MIT license found in the
```

Comment:
This line is required on every source code. It should only apply to generated code since we checked it in unless you explicitly exclude it using blacklist.
`# Copyright (c) Facebook, Inc. and its affiliates.`